src/main/java/uk/gov/justice/hmpps/probationteams/dto/LocalDeliveryUnit2Dto.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.hmpps.probationteams.dto
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
+import javax.validation.constraints.Email
+import javax.validation.constraints.NotBlank
+
+@ApiModel(description = "Local Delivery Unit")
+data class LocalDeliveryUnit2Dto @JsonCreator constructor(
+        @ApiModelProperty(required = true, value = "Probation Area code", position = 1, example = "NO2")
+        @JsonProperty("probationAreaCode")
+        val probationAreaCode: @NotBlank String,
+
+        @ApiModelProperty(required = true, value = "Local Delivery Unit code", position = 1, example = "NO2SUK")
```

Comment:
could potentially add `springfox-bean-validators` plugin which pulls required-ness from @NotBlank annotations (and supports other validations) meaning you don't need to repeat in swagger docs 

Guess max string length and things like that could be enforced if appropriate too.

Reply:
I didn't know about that.  Separate PR? Tech debt?